menu.js

```diff
@@ -0,0 +1,22 @@
+const closeMenu = () => {
+  document.querySelector('header').classList.remove('mobile-menu');
+  document.querySelector('.menu-btn').classList.remove('visible');
+  document.querySelector('nav > ul').classList.remove('mobile-nav');
+  document.querySelector('nav').classList.remove('nav-strech');
+  document.querySelector('.header > div').classList.remove('visible');
+};
```

Comment:
- [ ] When I click on the close button. the logo in the header disappears, please kindly fix this issue.

If you notice most websites have a logo so you can go back home page by clicking it 😃, that kind of UX convention make the navigation in the website way more pleasant and intuitive 

<div align="center">
<img src="https://user-images.githubusercontent.com/80895497/147292692-f068ebc6-50ed-41a1-8d5f-2161e4c3394b.PNG" alt="img" />
</div>

Reply:
You are very close to finishing your project @medaminedev66, but this bug 🐛 that @Hamzaoutdoors pointed still remains on your menu.js file, kindly check that when you click the menu button, the `class visible` it's added to your `class logo`, and when you click the close button the `class visible` it's not removed, that is why your logo is hidden.

- [ ] Please fix the bug on your logo behavior.